README.rst

```diff
@@ -57,23 +57,68 @@ our homepage_.
 .. |weblate| image:: https://weblate.bubu1.eu/widgets/searxng/-/searxng/svg-badge.svg
    :target: https://weblate.bubu1.eu/projects/searxng/
 
+Contact
+"""""""
+
+Matrix: `#searxng:matrix.org <https://matrix.to/#/#searxng:matrix.org>`_
+
+IRC:  `#searxng on libera.chat <https://web.libera.chat/?channel=#searxng>`_ which is bridged to Matrix.
 
-SearXNG is a fork of `searx`_:
 
-* `the differences with searx <https://github.com/searxng/searxng/issues/46>`_
-* `exhaustive changelog <https://github.com/searxng/searxng/wiki/Changes-from-version-1.0.0>`_
+Differences with searx
+""""""""""""""""""""""
+
+SearXNG is a fork of `searx`_.
+Here some of the changes:
 
 .. _searx: https://github.com/searx/searx
 
 
-Contact
-"""""""
+User experience
+~~~~~~~~~~~~~~~
 
-Matrix: `#searxng:matrix.org <https://matrix.to/#/#searxng:matrix.org>`_
+-  Huge update of the simple theme:
 
-IRC:  `#searxng on libera.chat <https://web.libera.chat/?channel=#searxng>`_ which is bridged to Matrix.
+   -  usable on desktop, tablet and mobile.
+   -  light and dark versions (you can choose in the preferences)
+   -  on the way to support RTL languages.
+
+-  the translations are up to date, you can contribute `here`_
+-  the preferences page has been updated:
+
+   -  you can see which engines is reliable or not. 
+   -  engines are grouped inside each tab.
+   -  each engine has a description
+
+-  it is easier to report a bug of an engine...
+-  ... but you can also disable the recording of the metrics on the
+   server.
+
+Setup
+~~~~~
+
+-  the docker image is built for the amd64, arm64 and arm/v7
+   architectures
```

Comment:
how is that different from searx? also capitalize Docker

Reply:
searx/searx supports only the AMD64 architecture.

Reply:
Right ... since this is about differences to searx/searx I think the point should be:

- the Docker image is now also built for arm64 and arm/v7 architectures

Reply:
Done